Source/Components/Touchpoints/LiveImages/ImageAnimationManager.swift

```diff
@@ -59,56 +57,95 @@ final class ImageAnimationManager: ImageAnimationManagerProtocol {
     }
     
     private func loadAnimatedImage(key: String) {
-        
+        changeState(to: .download)
         MLBusinessLiveImagesProvider.shared.getLiveImageData(from: key, completion: {
             [weak self] data in
             
             DispatchQueue.main.async {
                 if let dataEncoded = data?.base64EncodedString() {
+                    self?.changeState(to: .download_success)
                     self?.delegate?.setAnimatedImage(with: dataEncoded)
                 } else {
-                    self?.delegate?.changeState(to: .blocked)
+                    self?.changeState(to: .download_failed)
+                    self?.changeState(to: .blocked)
                 }
             }
         })
     }
     
-    func shouldHideAnimation(state: MLBusinessLiveImagesState) -> Bool {
-        return state != .playing
+    private func delayAnimation(delayTime: CGFloat = 1.5) {
+        self.delayWork = DispatchWorkItem(block: {
+            self.startAnimation()
+        })
+
+        if let work = delayWork {
+            DispatchQueue.main.asyncAfter(deadline: .now() + delayTime, execute: work)
+        }
     }
     
-    func stop(currentState: MLBusinessLiveImagesState) {
-        if currentState != .blocked {
+    private func startAnimation() {
+        changeState(to: .playing)
+        self.delegate?.transitionView()
+    }
+    
+    public init(imageProvider: MLBusinessImageProvider? = nil) {
+        self.imageProvider = imageProvider ?? MLBusinessURLImageProvider()
+    }
+
+    func update(coverMedia: MLBusinessLiveImagesModel?, cover: String?) {
+        delegate?.clear()
+        changeState(to: .init_multimedia)
+        if let coverMedia = coverMedia {
+            if let thumbnail = coverMedia.getThumbnail(), let url = coverMedia.getMediaLink() {
```

Comment:
Creo que acá podes juntar todos estos unwrap
```suggestion
        if let coverMedia = coverMedia,
        let thumbnail = coverMedia.getThumbnail(), 
        let url = coverMedia.getMediaLink() {
```